pyads/pyads_ex.py

```diff
@@ -71,7 +71,8 @@
             dll_path += "\\x64"
         dlldir_handle = os.add_dll_directory(dll_path)
     try:
-        _adsDLL = ctypes.WinDLL("TcAdsDll.dll")  # type: ignore
+        # _adsDLL = ctypes.WinDLL("TcAdsDll.dll")  # type: ignore
+        _adsDLL = ctypes.WinDLL("C:/TwinCAT/AdsApi/TcAdsDll/TcAdsDll.dll")
```

Comment:
It is never a good idea to hard-code the path to a library. You never know where the library is installed on the system. The purpose of the above line ist that the system finds it itself in the dedicated directories. So please revert this change.

Reply:
Absolutely right, this is a left-over change from my process of the getting the tests to run.